app/controllers/membership_applications_controller.rb

```diff
@@ -6,7 +6,10 @@ def new
   end
 
   def create
-    @membership_application = current_user.membership_applications.new(membership_application_params)
+    # must create the membership_application before using the policy to check the parameters. @see https://www.sitepoint.com/straightforward-rails-authorization-with-pundit/
+    @membership_application = current_user.membership_applications.new
+    @membership_application.update(permitted_attributes(@membership_application))
+
```

Comment:
I don't think this is necessary. You could probably authorize the class directly by calling 
```
authorize MembershipApplication
```
I think I didi it on the MembershipApplicationController#index